src/lib/shopping-fpti/event-handlers.js

```diff
@@ -8,16 +8,41 @@ function findConfigurationAttribute(config : Config, payload : Object = {}, attr
   return shopperConfig[attribName] || payload[attribName];
 }
 
+export const includedAttributes = [
+  // page view
+  'page_type',
+  'page_name',
+  'page_id',
+  'page_category_name',
+  'page_category_id',
+  'deal_id',
+  'deal_name',
+  'deal_value',
+  'search_results_count',
+  'cart_products',
+  // product view
+  'product_id',
+  'product_name',
+  'product_url',
+  'product_price',
+  'product_brand',
+  'product_category_name',
+  'product_category_id',
+  'product_discount',
+  // purchase
+  'amount',
+  // set properties
+  'currency',
+];
```

Comment:
This is totally fine .... but this seems like the perfect fit for a `@flow` type where we get this check for 'free'.

Reply:
Can we use the flow type and not affect the functionality of the code? Is flow types something we can only apply to tests?

Reply:
hmm yeah ..... the tests need to fail which will be complicated with a `@flow` type